data/text/default.ts

```diff
@@ -105,7 +105,7 @@ export const DefaultText: {[k: string]: DefaultText} = {
 		clearBoost: "  [POKEMON]'s stat changes were removed!",
 		clearBoostFromZEffect: "  [POKEMON] returned its decreased stats to normal using its Z-Power!",
 		invertBoost: "  [POKEMON]'s stat changes were inverted!",
-		clearAllBoost: "  All stat changes were eliminated!",
+		clearAllBoost: "  All stats changes were eliminated!",
```

Comment:
This won't be changed because it's text lifted directly from the game.

Reply:
Sorry